var/spack/repos/builtin/packages/care/package.py

```diff
@@ -25,9 +25,8 @@ class Care(CMakePackage, CudaPackage, ROCmPackage):
     variant('benchmarks', default=True, description='Build benchmarks.')
     variant('examples', default=True, description='Build examples.')
     variant('docs', default=False, description='Build documentation')
-    # TODO: figure out gtest dependency and then set this default True
-    # and remove the +tests conflict below.
     variant('tests', default=False, description='Build tests')
+    variant('loop_fuser', default=False, description='Enable loop fusion capability')
```

Comment:
is it possible to have this on by default with the hip or cuda variant?

Reply:
Not as far as I know. I tried to do

    isgpu = self.spec.variants['rocm'].value or self.spec.variants['cuda'].value
    variant('loop_fuser', default=isgpu, description='Enable loop fusion capability')

But things have not been initialized yet at this point, so no luck. I could certainly override the value in the cmake_args function below, but that is not ideal. My guess is that conditional defaults for variants would require a change to lib/spack/spack/variant.py . 

I might be wrong though; @becker33 would certainly know better

Reply:
We can't support conditional variants until we remove the default concretizer and convert fully to the new concretizer. It is on our roadmap to eventually support, but there's some work that needs to happen before we get there.